aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -480,17 +544,40 @@ std::tuple<Tensor, Tensor> flash_attention_helper_dense_unpacked(
 
   return std::tuple<Tensor, Tensor>(attention, Tensor());
 }
+std::tuple<Tensor, Tensor> mem_eff_helper(
+    const Tensor& query,
+    const Tensor& key,
+    const Tensor& value){
+  // Query -> Query(Batch x Q_seq_len x Num_heads x Dim_per_head)
+  // Key   -> Key(Batch x KV_seq_len x Num_heads x Dim_per_head)
+  // Value -> Value(Batch x KV_seq_len x  Num_heads x Dim_per_head)
```

Comment:
very minimal but still inlining this to the case block would be a little too big